modules/actor/actor-wfrp4e.js

```diff
@@ -3885,8 +3887,8 @@ export default class ActorWfrp4e extends Actor {
     return (this.itemCategories || this.itemTypes)[type]
   }
 
-  clearOpposed() {
-    return this.update({ "flags.-=oppose": null })
+  async clearOpposed() {
+    await this.update({ "flags.-=oppose": null })
```

Comment:
I believe this is the only unresolved comment left

Reply:
I would like to keep this one as it is

Reply:
Why? There's no difference?

Reply:
Well, there is a difference, not returning the promise prevents it being used in the calling context, which may be useful.

Reply:
if i understand correctly the sugar-syntax around async/await - under the hood it does return implicitely a promise. The only problem with using await instead of .then(whatever) is that in await you cannot handle reject other way than throwing an error. It's more that i want to await this method inside all the 'handle opposed' chain, and i think it makes it a bit more readable when you await explicit async function instead of function that simply returns Promise (which would also work, because you can await any function, even non-async).

Reply:
It returns a promise regardless yeah, but the promise's resolved value from `update` is discarded and now just returns `undefined`